Example/Okta/Okta-PasswordFlow.plist

```diff
@@ -5,12 +5,12 @@
 	<key>scopes</key>
```

Comment:
@nbarbettini - Given that this PR will introduce a breaking change, I'd propose we remove ROPF support before releasing `2.0.0`.

Therefore, this file and quite a bit of custom logic can be removed.

Reply:
+ @robertjd

Reply:
I agree. @allebedew let's remove the `OktaAuth.login(username: string, password: string)` functionality. Only keep the browser sign-in flow. We will add that functionality back via #82.

Reply:
OK, removing native flow...

Reply:
Will post as another pull-request